packages/server/__tests__/inviteToTeam.test.ts

```diff
@@ -28,43 +27,7 @@ test('Invite to team, works for ordinary email domain', async () => {
     data: {
       inviteToTeam: {
         error: null,
-        invitees: ['foo@example.com', inviteeEmail]
-      }
-    }
-  })
-})
-
-test('Invite to team, deny for untrusted domain', async () => {
-  const {userId, authToken} = await signUp()
-  const {email: inviteeEmail} = await signUp()
-
-  const teamId = (await getUserTeams(userId))[0].id
-  const inviteToTeam = await sendPublic({
-    query: `
-      mutation InviteToTeam($teamId: ID!, $invitees: [Email!]!) {
-        inviteToTeam(teamId: $teamId, invitees: $invitees) {
-          error {
-            title
-            message
-          }
-          invitees
-        }
-      }
-    `,
-    variables: {
-      teamId,
-      invitees: ['foo@qq.com', inviteeEmail]
-    },
-    authToken
-  })
-
-  expect(inviteToTeam).toMatchObject({
-    data: {
-      inviteToTeam: {
-        error: {
-          message: 'Cannot invite by email. Try using invite link',
```

Comment:
We're no longer returning this error message from inviteToTeamHelper so I think it's safe to remove